ArkEcosystem/laravel-foundation
# refactor: update sidebar styling

## Summary

https://app.clickup.com/t/26gucbv

To test: 

- checkout `refactor/use-foundation-components` on ark.dev (https://github.com/ArkEcosystem/docs/pull/242)
- use `refactor/update-sidebar-styling-tutorial` for foundation
- clear cache
- yarn install & yarn prod
- sidebars should match design







## Checklist
